### PR TITLE
Update Readme to match the template

### PR DESCRIPTION
**Pull request by GomathiselviS**
##### SUMMARY

Refer: https://issues.redhat.com/browse/ACA-1749
This PR updates the README doc to match the [template](https://access.redhat.com/articles/7068606)

##### ISSUE TYPE

- Docs Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,11 +53,13 @@ See the complete list of collection content in the [Plugin Index](https://ansibl
 
 <!--end collection content-->
 
-## Installing this collection
+## Installation
 
-You can install the AWS collection with the Ansible Galaxy CLI:
+Before using this collection, you need to install it with the Ansible Galaxy command-line tool:
```
Comment (mandar242) [inline README.md:58]:
```suggestion
The amazon.aws collection can be installed with Ansible Galaxy command-line tool:
```

File: README.md [hunk 2]
```diff
@@ -70,6 +84,12 @@ collections:
     version: 3.1.1
 ```
 
+or using the ansibble-galaxy command as follows
```
Comment (mandar242) [inline README.md:87]:
```suggestion
A specific version of amazon.aws collection can be installed using the ansible-galaxy command as follows
```

File: README.md [hunk 3]
```diff
@@ -97,48 +121,70 @@ You can either call modules by their Fully Qualified Collection Name (FQCN), suc
...
-## Roadmap
+- Libera.Chat IRC - the ``#ansible-aws`` [irc.libera.chat](https://libera.chat/) channel
 
-<!-- Optional. Include the roadmap for this collection, and the proposed release/versioning strategy so users can anticipate the upgrade/update cycle. -->
+## Release notes
+
+See the [rendered changelog](https://ansible-collections.github.io/amazon.aws/branch/main/collections/amazon/aws/docsite/CHANGELOG.html) or the [raw generated changelog](https://github.com/ansible-collections/amazon.aws/tree/main/CHANGELOG.rst).
 
-## More information
+## Related Information
```
Comment (mandar242) [inline README.md:179]: maybe something like this?
```suggestion
## Useful Resources
```

**Review by mandar242 — APPROVED** [review]
Overall LGTM, matches https://access.redhat.com/articles/7068606
put some comments, just suggestions though. If we have to stick with content of article, then can be ignored.

File: README.md (hunk 2 above)
Comment (gravesm) [inline README.md:87]:
```suggestion
or using the ansible-galaxy command as follows
```

or use @mandar242's suggestion

Comment (GomathiselviS) [reply README.md:87]: The line - "A specific version of amazon.aws collection can be installed" , is already mentioned previously. I will fix typo.

**Review by gravesm — APPROVED** [review]
Small typo, otherwise looks good.

File: README.md (hunk 3 above)
Comment (GomathiselviS) [reply README.md:179]: I have used what is given in the template. I think that is the requirement.

**Comment by GomathiselviS** [comment]
regate

**Pull request merged**